build_and_release.rb

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/ruby
+
+# This script builds the code, overriding the version number of the pom with
+# one of the form MAJOR.MINOR.BUILD. The major and minor version numbers are
+# taken from the pom file (see the BLUEFLOOD_VERSION maven parameter). The
+# build number should be passed in as the first and only command line argument
+# to the script. The script will then turn a snapshot version string, such as
+# "2.0-SNAPSHOT", into something suitable for a release build, like "2.0.36".
+#
+# Running maven from the command line (e.g. `mvn package`), instead of the
+# script, will still produce artifacts with the SNAPSHOT version string.
+
+require 'rexml/document'
+
+
+def get_version_number(build_number)
+  pom = REXML::Document.new(File.new("pom.xml"))
+  version = pom.elements["project/properties/BLUEFLOOD_VERSION"].text
+
+  if version.end_with? '${VERSION_SUFFIX}'
```

Comment:
@ChandraAddala I think I understand what you mean. You're talking about the case when someone manually changes the value of `BLUEFLOOD_VERSION` in the pom itself, right? So something like this:
```xml
<BLUEFLOOD_VERSION>2.0.1</BLUEFLOOD_VERSION>
```
In that case, it would be good for the build script to check the format of the variable. Is that what you're driving at?